javax-money/pom.xml

```diff
@@ -0,0 +1,119 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <!-- This module was also published with a richer model, Gradle metadata,  -->
+    <!-- which should be used instead. Do not delete the following line which  -->
+    <!-- is to indicate to Gradle or any Gradle module metadata file consumer  -->
+    <!-- that they should prefer consuming it instead. -->
+    <!-- do_not_remove: published-with-gradle-metadata -->
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.fasterxml.jackson.datatype</groupId>
+        <artifactId>jackson-datatypes-misc-parent</artifactId>
+        <version>2.19.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>jackson-datatype-money</artifactId>
+    <name>Jackson datatype: javax-money</name>
+    <packaging>jar</packaging>
+    <version>2.19.0-SNAPSHOT</version>
+    <description>Support for datatypes of Money API spec from JSR 354 (https://javamoney.github.io/api.html)
+    </description>
+    <url>https://github.com/FasterXML/jackson-datatypes-misc</url>
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://opensource.org/licenses/MIT</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+    <properties>
+        <!-- Generate PackageVersion.java into this directory. -->
+        <packageVersion.dir>com/fasterxml/jackson/datatype/money</packageVersion.dir>
+        <packageVersion.package>${project.groupId}.money</packageVersion.package>
+        <slf4j.version>2.0.6</slf4j.version>
+        <junit-jupiter.version>5.9.2</junit-jupiter.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>javax.money</groupId>
+            <artifactId>money-api</artifactId>
+            <version>1.1</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.javamoney.moneta</groupId>
+            <artifactId>moneta-core</artifactId>
+            <version>1.4.2</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apiguardian</groupId>
+            <artifactId>apiguardian-api</artifactId>
+            <version>1.1.2</version>
```

Comment:
Also, this... if these are annotations, should probably use scope of `provided`?

At least it would seem like this would not be typical `compile` dependency for actual functionality?